_data/collection-index.yml

```
@@ -488,3 +488,8 @@
   contact: https://github.com/earthly/devcontainer-features/issues
   repository: https://github.com/earthly/devcontainer-features
   ociRefernce: ghcr.io/earthly/devcontainer-features
```

```suggestion
  ociReference: ghcr.io/earthly/devcontainer-features
```

Looks like the previous entry has a typo with  `ociReference`. @christian-bromann if possible, can you help fix that as well?